ContactUs.html

```diff
@@ -10,8 +10,13 @@
 <meta name="google-signin-client_id" content="365679910149-388t7l2qc3kflnj52f01mk4bku5ffiis.apps.googleusercontent.com">
 <script src="https://apis.google.com/js/platform.js" async defer></script>
 <title>Contact Us</title>
-<script src="https://code.jquery.com/jquery-3.3.1.js"></script>
-<script src="js/NavOpacity.js"></script>
+<style type="text/css">
+.i {position: 600px;}
```

Comment:
Style: We shouldn't put the "position" on the same lines as the braces. It makes fixing merge conflicts harder.
Try something like:
```
.i {
position: 600px;
}
```